### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_View/Select_Object Information: Issues displayed in assessments results dialog are defined as table instead of list items

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19866]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Github Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Benchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG4.2.1; #Object Information;

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Repro Steps:

1. Turn on Narrator/NVDA
2. Install Azure data studio app and install Azure SQL migration extension
3. Connect to SQL server, tab till respective server, right click and select manage
4. Tab till Azure SQL Migration in sitemap and press enter key
5. Tab till migrate to azure sql and press enter
6. Navigate to databases grid in "Databases for assessment", select few databases and click next.
7. In Assessment results and recommendations step, select a azure sql target and navigate to "View/select" button and press enter
8. After selecting a database in "Assessment results" dialog, issues will be displayed.
9. Verify that issues displayed in assessments results dialog are defined as table

### Actual Result:
Issues displayed in assessments results dialog are defined as table instead of list items

### Expected Result:
Issues displayed in assessments results dialog should be defined as list items instead of giving a empty column headers and empty column 1.

### Attachment:
![table_](https://user-images.githubusercontent.com/83570638/176417332-ce5f555d-0c6d-4d1e-bc0a-38ca276a44a7.gif)

**Comment by aasimkhan30** [comment]
@brian-harris, can you please change it to a list component?
Example:
https://github.com/microsoft/azuredatastudio/blob/4096037167a7fea5ab3c4efcd6a041dfcfa687d4/extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts#L136

It looks like this
![image](https://user-images.githubusercontent.com/6816294/182723176-3bca0dd4-f2c5-492b-9cc3-2d2b64e11c33.png)

**Comment by AkshayMata** [comment]
@aasimkhan30 please re-assign yourself. The table component was used because we have icons for blocking scenarios and list view doesn't have support. 

we also discussed creating a single column table with flex container in each cell to add the icons, but we agreed that adding icons to list view will be the better solution going forward. @aasimkhan30 to add support for this

**Comment by VijayaBharathyA** [comment]
#Closed;
Verified the issue in below Environment:
Issue is Fixed
Version: 1.41.1 (user setup)
![Screenshot (29)](https://user-images.githubusercontent.com/100189407/218051937-94d25ff3-7d8d-4640-a191-2bdcde2c4e35.png)
